csrankings-s.csv

```diff
@@ -176,7 +176,7 @@ Sameem Abdul Kareem,University of Malaya,https://umexpert.um.edu.my/sameem,Rl2eb
 Sameem Binti Abdul Kareem,University of Malaya,https://umexpert.um.edu.my/sameem,Rl2ebS4AAAAJ
 Sameer G. Kulkarni,IIT Gandhinagar,https://iitgn.ac.in/faculty/cse/sameer,CnPfKYUAAAAJ
 Sameer Patil,University of Utah,https://www.sameerpatil.com,JGEniHoAAAAJ
-Sameer Singh 0001,Univ. of California - Irvine,http://sameersingh.org,2RFiH8oAAAAJ
+Sameer Singh 0001,Univ. of California - Irvine,http://sameersingh.org,-hGZC54AAAAJ

```

Comment:
LinkedIn says CTO of a company; likely incompatible with full-time appointment.
https://www.linkedin.com/in/sameersingh/

Reply:
@sameersingh Please confirm or deny >= 75% current appointment in academia.

Reply:
No response; remove for now. Thanks.